client/src/main/java/org/spine3/base/Failures.java

```diff
@@ -38,4 +45,46 @@ public static FailureId generateId() {
                         .setUuid(value)
                         .build();
     }
+
+    /**
+     * Extracts the message from the passed {@code Failure} instance.
+     *
+     * @param failure a failure to extract a message from
+     * @param <M>     a type of the failure message
+     * @return an unpacked message
+     */
+    public static <M extends Message> M getMessage(Failure failure) {
+        checkNotNull(failure);
+        final M result = AnyPacker.unpack(failure.getMessage());
```

Comment:
It would be good to have `pack/unpack` invoked in a symmetric way. Now `pack()` is statically imported, while `unpack()` is not.